plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -60,9 +62,9 @@ var attributesRenamingForTrace = map[string]string{
 	attr.AWSRemoteTarget: attr.AWSRemoteResourceIdentifier,
 }
 
-var copyMapForMetric = map[string]string{
+var resourceAttributesToMetric = map[string]string{
 	// these kubernetes resource attributes are set by the openTelemtry operator
```

Comment:
nit: Since you're fixing another typo from the line below.
```suggestion
	// these kubernetes resource attributes are set by the opentelemetry operator
```

Reply:
Updated